frontend/www/js/omegaup/components/homepage/Sponsors.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div class="row p-4 text-center justify-content-around align-items-center">
     <h3 class="col-12 mb-3 display-4">{{ title }}</h3>
-    <div v-for="logo in logos" :key="logo.href" class="p-3 mt-2 mt-md-0">
+    <div v-for="logo in logos" :key="logo.href" class="p-3 mt-2 mt-md-0 sponsor-logo-container">
       <a :href="logo.href" target="_blank">
-        <img :class="logo.class" :src="logo.src" :alt="logo.alt" />
+        <img :class="logo.class" :src="logo.src" :alt="logo.alt" :style="logo.style || ''" />
```

Comment:
It wouldn’t be used here either:
```suggestion
        <img :class="logo.class" :src="logo.src" :alt="logo.alt" />
```

Reply:
I see!